src/network/networkd-dhcp-server.c

```diff
@@ -2,6 +2,7 @@
 
```

Comment:
btw, could you prefix the commit msg with the subsystem you touch? i.e. with `networkd: `. See the "Commiting to git" part of https://systemd.io/CODING_STYLE/#committing-to-git